Wordle.Api/Wordle.Api/Identity/Roles.cs

```diff
@@ -2,4 +2,5 @@
 public static class Roles
 {
     public const string Admin = "Admin";
```

Comment:
Just to clarify, awesome would technically be the one with the role MasterOfTheUniverse?

Reply:
Yes. I added it in a weird way so I'm going to update the way we add the claim so its easier to see.